javasysmon.iml

```diff
@@ -4,10 +4,10 @@
     <output url="file://$MODULE_DIR$/target/classes" />
     <output-test url="file://$MODULE_DIR$/target/test-classes" />
     <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/lib/native" type="java-resource" />
```

Comment:
We should not include this change that is only reordering a liner. Maybe IntelliJ automatically modified your file.